api/app/src/routes/webhook/fitbit.ts

```diff
@@ -0,0 +1,51 @@
+import { Request, Response } from "express";
+import Router from "express-promise-router";
+import { asyncHandler } from "../util";
+import { processData } from "../../command/webhook/fitbit";
+
+const routes = Router();
+
+// This value is provided on the dev.fitbit dashboard when you update or craete a new webhook subscriber.
+const FITBIT_VERIFICATION_CODE = "";
+
+/** ---------------------------------------------------------------------------
+ * GET /webhook/fitbit
+ *
+ * Verify Fitbit WH subscriber. Receives a verification notification and sends back a verification post request.
+ *
+ */
+routes.get(
+  "/",
+  asyncHandler(async (req: Request, res: Response) => {
+    if (req.query.verify) {
+      console.log(
+        "Verifying the subscriber. Make sure to use the verification code provided in the dev.fitbit dashboard!"
+      );
+      if (req.query.verify === FITBIT_VERIFICATION_CODE) {
+        console.log("Received correct verification code.");
+        return res.sendStatus(204);
+      } else {
+        console.log("Incorrect verification code detected!");
+        return res.sendStatus(404);
+      }
+    } else {
+      return res.sendStatus(200);
+    }
+  })
+);
+
+/** ---------------------------------------------------------------------------
+ * POST /webhook/fitbit
+ *
+ * Receive fitbit data for all data types for the specified user ID
+ *
+ */
+routes.post(
+  "/",
+  asyncHandler(async (req: Request, res: Response) => {
+    processData(req.body);
```

Comment:
Forgot to mention earlier: don't we want to use a zod schema to validate the input/body?

Currently, body is `any` which is why the compiler is not complaining and allowing it to be passed as a parameter to `processData`, but it doesn't mean body is an array, nor that each element matches the shape espected on `processData`.

If we don't validate and Fitbit changes the API, for example, we might spend hours trying to understand why something is failing, instead of getting a validation error - easier to identify.